Sources/SwiftDocC/Infrastructure/DocumentationContext.swift

```diff
@@ -2348,8 +2348,8 @@ public class DocumentationContext: DocumentationContextDataProviderDelegate {
         return automaticallyCuratedSymbols
     }
 
-    private func addOverloadGroupReferences(overloadGroups: [String: [String]]) {
-        guard FeatureFlags.current.isExperimentalOverloadedSymbolPresentationEnabled else {
+    private func addOverloadGroupReferences(from bundle: DocumentationBundle, overloadGroups: [String: [String]]) {
+        guard bundle.info.computedFeatureFlags.experimentalOverloadedSymbolPresentationEnabled else {
```

Comment:
I'm a bit concerned that we're making the call sites for feature flags less readable using this new pattern. Also we're applying this new style only in a few places; there are other feature flags elsewhere in the code that continue to use the  `FeatureFlags.current...` pattern. The inconsistent style might confuse developers working near the call sites or who add new flags in the future.

Reply:
maybe it would be more readable to have something like 
```swift
bundle.info.computedFeatureFlags.contains(.overloadSymbolPresentation)
```

Reply:
We could also use the decoded feature flags to populate the main `FeatureFlags`. That way we only need one way to _check_ a feature flag.

Reply:
The issue i can imagine happening with modifying the global feature flags struct with the parsed feature flags is that we still technically support processing multiple bundles at the same time. If only one of these bundles enables a feature flag, should that enable it for all of the bundles being processed?

Reply:
Also, to address Pat's suggestion in the review comment: AFAIK, there is no concept of "current documentation bundle" that i can quickly reference for a lookup like this, especially not from the perspective of a global singleton like the feature flags list. Introducing a new concept like that feels like a bigger refactor than i want to commit to for a PR like this. I could see writing an accessor on the global feature flags struct that takes an optional documentation bundle, but that still requires changing all the use sites to match the new accessor, and requires giving a bundle instance to that function. Are we okay with writing a widespread change like that?

Reply:
Multiple bundles is a theoretical possibility that doesn't work in practice because DocC actually only builds the "first" bundle and ignores the other. 

Since building multiple bundles has never been supported I plan on cleaning up the code so that the internal model matches the user facing capabilities (one bundle per build). I just haven't gotten around to it yet.

I would say that global updating the shared feature flags with the values parsed from the Info.plist is better than passing the bundle around everywhere.